src/docs/operator-guides/how-to-edit-pages/index.md

```diff
@@ -0,0 +1,295 @@
+
+## Edit Sections
+
+At any point you can edit your default pages by modifying the content
+within the default pages' Sections. Press on the Section ID to start
+editing sections. In order to save any changes to a section you need to
+click the “Save changes” button at the bottom of the page. We recommend
+editing a section at a time, to make it easy to identify errors of
+missing fields that would prevent you from saving changes. Bear in mind
+that the "Save changes" button only gets enabled when there are changes
+to be saved. I.e. if you don’t make changes to a section, or you make
+changes and then remove the changes the button will be disabled.
+
+As soon as you save the changes of the page, you can click the link to
+visit the page to check the new changes or, if you already have the page
+open somewhere, reload the page to see the changes. In the demo
+environment the changes will be reflected immediately. In the production
+environment it will take 5 minutes to see the changes reflected. This is
+due to the page’s
+[cache](<https://en.wikipedia.org/wiki/Cache_(computing)>).
+
+### Reorder Sections and Blocks
+
+You can alter the order of the sections within the landing page or the
+blocks within a section by dragging and dropping the elements. Make sure
+to save your changes after altering the section content or changing the
+order of the elements, otherwise they won’t take effect on your live
+landing page.
+
+![reordering sections](/edit-sections.gif 'drag and drop to reorder a Section')
+
+### Remove Sections or Blocks
+
+At any point you can remove your landing page sections and blocks by
+clicking on the three-dot menu on the top right corner of the section
+card. Bear in mind that even though the action is technically
+irreversible, you still need to save your landing page for the changes
+to take effect. If you don’t save your landing page changes the delete
+action will not be completed.
+
+### Can I edit the Footer or the Top bar?
+
+The footer or Top bar cannot be edited using Pages. Changing their
+content requires changing your marketplace code directly. Contact your
+developer if you need help with changes or follow our
+[developer documentation](https://www.sharetribe.com/docs/ftw/how-to-add-static-pages-in-ftw/)
+to learn more about editing the code of content pages.
+
+## Publish your landing page changes to Production
+
+To see your changes in your live marketplace, you must copy the changes you made to your production environment from your development environment. Click
+the “Copy to production…” button and then select the specific pages that
+you want to copy. This will transfer all the content from the current
+environment to your production environment.
+
+The changes that you make to production will be available a few minutes
+after copying them to production. You do not need to save them again in
+your production environment.
+
+![copy-to-production](/copy-to-production.jpg 'Copy changes to production')
```

Comment:
For some reason the layout of the UI seems broken in this screenshot. There are weird gaps between the items. Can you try taking the screenshot again? I can't reproduce the layout issue in the real Console.